src/engine/Engine.cpp

```diff
@@ -1821,6 +1824,27 @@ List<unsigned> Engine::getInputVariables() const
     return _preprocessedQuery.getInputVariables();
 }
 
+void Engine::performSimulation()
+{
+    // outer vector is for neuron
+    // inner vector is for simulation value
+    std::vector<std::vector<double>> simulations;
```

Comment:
Could you try to use the Vector class instead of std::vector to be consistent with the rest of the project?
Hopefully a search and replace would do.

Reply:
Like in https://github.com/NeuralNetworkVerification/Marabou/blob/8d777410f270af3bfdca747ed7f5acf7f3feed53/src/engine/BoundManager.h#L41

Vector is just a wrapper class around std::vector

Reply:
done.